### PR TITLE
chore: use exm aether (2.0.0)

### PR DESCRIPTION
**Pull request by lordmallam**

### CODE REVIEW

**Comment by obdulia-losantos** [comment]
We cannot merge this PR till aether `release-2.0` branch is approved and merge into the `develop` one.

**Comment by shawnsarwar** [comment]
@lordmallam @obdulia-losantos Shall we merge 2.0 back -> develop? I think it's about time.

**Comment by obdulia-losantos** [comment]
We cannot do that till @MrPink we can ensure that the alpha server is ready with the redis instance...

File: docker-compose-base.yml [hunk 1]
```diff
@@ -111,6 +121,26 @@ services:
...
+  # ---------------------------------
+
+  exm-base:
+    image: ehealthafrica/aether-exm:${AETHER_VERSION:-alpha}
+    environment:
+      DEFAULT_REALM: ${DEFAULT_REALM}
+      REALM_COOKIE: ${REALM_COOKIE}
+
+      # use internal url, this happens behind the curtains
+      AETHER_KERNEL_URL: http://aether.local/kernel
```
Comment (obdulia-losantos) [inline docker-compose-base.yml:146]:
```suggestion
      AETHER_KERNEL_URL: http://kernel:8100
```

**Pull request merged**